docs/incubator/index.yaml

```diff
@@ -3,7 +3,7 @@ entries:
   albatross:
   - apiVersion: v1
     appVersion: "1.0"
-    created: "2021-10-22T12:49:48.145174+05:30"
```

Comment:
Can you revert all the changes and keep the changes present only for envoy-lb-operator

Reply:
@punit-kulal  I have made the suggested reverts on the last commit. please take a look.